src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -225,7 +229,7 @@ using LayerTypes = boost::variant<
     Sequential<arma::mat, arma::mat, true>*,
     Subview<arma::mat, arma::mat>*,
     // VRClassReward<arma::mat, arma::mat>*,
-    VirtualBatchNorm<arma::mat, arma::mat>*,
+    MoreTypes,
```

Comment:
Do you think we should move more layers to ``MoreTypes`` as this might open up the possibility to add more custom layers.

Reply:
Yes you are quite correct. We can move them. Just let me know how many we should move. If we move all of them then it will be 50 * 50 = 2500 layers.

Reply:
I guess for now we can move 10 or so into ``MoreTypes``.

Reply:
Done.

Reply:
Great, thanks, let's wait for the build jobs.